mappings/net/minecraft/server/world/chunk/light/ServerLightingProvider.mapping

```diff
@@ -1,2 +1,5 @@
 CLASS uu net/minecraft/server/world/chunk/light/ServerLightingProvider
 	METHOD a queueLightCheck (Let;)V
+		ARG 1 blockPos
```

Comment:
Probably should be just `pos` -- most other code seems to follow that.

Reply:
I will change that together with the next time I push my changes